### PR TITLE
Port in use message

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

**Comment by hackaugusto** [comment]
This is not stopping the exception stack trace from being printed, which is a bit annoying since the exception is being handled. @andrevmatos do you have a suggestion on how to fix this?

File: raiden/ui/cli.py [hunk 1]
```diff
@@ -1055,12 +1061,19 @@ def stop_task(task):
                 finally:
                     task.get()  # re-raise
 
-            stop_tasks = [gevent.spawn(stop_task, task) for task in tasks]
-            gevent.joinall(
-                stop_tasks,
-                app_.config.get('shutdown_timeout', DEFAULT_SHUTDOWN_TIMEOUT),
-                raise_error=True,
-            )
+            try:
```
Comment (andrevmatos) [inline raiden/ui/cli.py]: Don't handle here, this is exactly the point where any unhandled exceptions **until this point** should re-raise

File: raiden/ui/cli.py [hunk 2]
```diff
@@ -978,10 +984,10 @@ def _run_app(self):
...
                 api_server.start(api_host, api_port)
             except APIServerPortInUseError:
-                print(
-                    'ERROR: API Address %s:%s is in use. '
-                    'Use --api-address <host:port> to specify port to listen on.' %
-                    (api_host, api_port),
+                click.secho(
+                    f'ERROR: API Address {api_host}:{api_port} is in use. '
+                    f'Use --api-address <host:port> to specify a different port.',
+                    fg='red',
```
Comment (andrevmatos) [inline raiden/ui/cli.py]: At this point, the `api_server.start` exception is being handled and `api_server` was not added yet to the `tasks` list. In the line below, you should clean the already started stasks (as done by that `stop_task`+`joinall` in the end of this function), and **then** exit.

File: raiden/ui/cli.py [hunk 3]
```diff
@@ -770,6 +773,9 @@ def run_app(
 
     try:
         raiden_app.start()
+    except RuntimeError as e:
+        click.secho(f'FATAL: {e}', fg='red')
+        sys.exit(1)
```
Comment (andrevmatos) [inline raiden/ui/cli.py]: [Same here](https://github.com/raiden-network/raiden/pull/2365/files#r215314556)

File: raiden/ui/cli.py (hunk 1 above)
Comment (hackaugusto) [reply raiden/ui/cli.py]: the exception was not handled until here, where do you want it to be handled?

Comment (andrevmatos) [reply raiden/ui/cli.py]: [Here](https://github.com/raiden-network/raiden/pull/2365#discussion_r215314556)

Comment (hackaugusto) [reply raiden/ui/cli.py]: done by https://github.com/raiden-network/raiden/pull/2446

**Pull request merged**